# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1324512821 18000
#      Wed Dec 21 19:13:41 2011 -0500
# Branch stable
# Node ID 9c1f630dc03d3f7e9c01abd8d79720073a6b2a58
# Parent  8782d41893c6b95f944812ffe95c4f9bfc8c6aab
polar.m: Add clf() to demos (bug 35142).

diff --git a/scripts/plot/polar.m b/scripts/plot/polar.m
--- a/scripts/plot/polar.m
+++ b/scripts/plot/polar.m
@@ -212,17 +212,19 @@ function retval = __plr2__ (h, theta, rh
   else
     error ("__plr2__: invalid data for plotting");
   endif
 
 endfunction
 
 
 %!demo
+%! clf
 %! theta = linspace (0, 2*pi, 1000);
 %! rho = sin (7*theta);
 %! polar (theta, rho);
 
 %!demo
+%! clf
 %! theta = linspace (0, 10*pi, 1000);
 %! rho = sin (5/4*theta);
 %! polar (theta, rho);
 
